source/dom/dom.css

```diff
@@ -134,3 +134,9 @@ body .enyo-fullscreen {
 	height: 100% !important;
 	box-sizing: border-box !important;
 }
+
+/* Accessibility CSS */
+*[tabindex] {
+	/* remove outline in case dom has tabindex attribute */
+	outline: none;
```

Comment:
Don't we need to put -webkit-tap-highlight to disable browser's default highlight when tap?